### PR TITLE
[TIMOB-25675] : iOS11 Button in navigation-bar remains in pressed state when opening new window

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-25675

### CODE REVIEW

File: iphone/Classes/TiUIiOSNavWindowProxy.m [hunk 1]
```diff
@@ -181,6 +181,11 @@ - (void)windowClosing:(TiWindowProxy *)window animated:(BOOL)animated
 
 - (void)navigationController:(UINavigationController *)navigationController willShowViewController:(UIViewController *)viewController animated:(BOOL)animated
 {
+  if ([TiUtils isIOSVersionOrGreater:@"11.2"]) {
+    navigationController.navigationBar.tintAdjustmentMode = UIViewTintAdjustmentModeNormal;
+    navigationController.navigationBar.tintAdjustmentMode = UIViewTintAdjustmentModeAutomatic;
```
Comment (hansemannn) [inline iphone/Classes/TiUIiOSNavWindowProxy.m]: Both? That doesn‘t seem righr.

Comment (vijaysingh-axway) [reply iphone/Classes/TiUIiOSNavWindowProxy.m]: Yes. If you see the discussion at https://stackoverflow.com/questions/47754472/ios-uinavigationbar-button-remains-faded-after-segue-back, the same is suggested.

Comment (hansemannn) [reply iphone/Classes/TiUIiOSNavWindowProxy.m]: Weird, but valid. Can you file a radar for this as well?

**Review by hansemannn — APPROVED** [review]

File: iphone/Classes/TiUIiOSNavWindowProxy.m (hunk 1 above)
Comment (vijaysingh-axway) [reply iphone/Classes/TiUIiOSNavWindowProxy.m]: Yes. I'll file bug to apple.

**Comment by ewieberappc** [comment]
FR Passed on 7_0_X branch

**Pull request merged**